recipe/meta.yaml

```diff
@@ -74,14 +69,13 @@ outputs:
         - mpl_toolkits
         - pylab
       requires:
-        - pytest
```

Comment:
There are some tests in lib/matplotlib we could run

Reply:
I looked into it but we would have to include lib/matplotlib/tests/baseline_images which would make the package huge.

Reply:
I'm starting to think that we need an alternative way of getting test resources other tha just embedding files into packages...

Reply:
An alternative could be adding an additional output. That's something we should discuss eventually.